joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/datastructures/RubyProgramSummary.scala

```diff
@@ -30,10 +31,17 @@ case class RubyMethod(
   baseTypeFullName: Option[String]
 ) extends MethodLike
 
-case class RubyField(name: String, typeName: String) extends FieldLike
+object RubyMethod {
+  implicit val rubyMethodRwJson: ReadWriter[RubyMethod] = readwriter[ujson.Value].bimap[RubyMethod](
+    x => ujson.Obj("name" -> x.name),
+    json => RubyMethod(json("name").str, List.empty, "", Option.empty)
```

Comment:
Note `Option.empty` should be the parent `RubyType`. Thinking about this now, we should probably write a `ReadWriter` for `RubyType` that can re-map the `RubyTypes.methods` field with `baseTypeFullName` to `RubyType.name`
```suggestion
    json => RubyMethod(json("name").str, List.empty, Defines.Any, Option.empty)
```